x-pack/plugins/reporting/server/index.ts

```diff
@@ -4,11 +4,66 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { Legacy } from 'kibana';
```

Comment:
Hopefully this will be ok, but it will probably go away once Reporting routes are migrated to NP.